configure.ac

```diff
@@ -183,6 +183,8 @@ AS_IF([test -n "$GCC"], [
 	WARNINGS="$WARNINGS -Werror"
     ])
 
+    CFLAGS="$CFLAGS $WARNINGS"
```

Comment:
Hmm, these really should be substituted in the makefile instead... I also notice I'm not including CPPFLAGS in the makefile, either... :/